students/toddschultz/session08/test_circle.py

```diff
@@ -0,0 +1,88 @@
+from circle import Circle
+
+
+def test_radius():
+	a = Circle(4)
+	assert a.radius is 4
+
+def test_diameter_1():
+	b = Circle(4)
+	assert b.diameter is 8
+
+def test_diameter_2():
+	c = Circle(4)
+	c.diameter = 10
+	assert c.diameter is 10
+
+def test_add():
+	e = Circle(4)
+	f = Circle(6)
+	assert e.radius + f.radius is 10
+	assert e.diameter + f.diameter is 20
```

Comment:
`is` is used for object comparison; for example `is True` `is None` etc

here you should use `==` operators